game_eggs/steamcmd_servers/night_of_the_dead/Night of the dead.json

```diff
@@ -0,0 +1,31 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-07-12T22:41:37+02:00",
+    "name": "NOTD",
+    "author": "info@prospectbyte.nl",
+    "description": "Server egg for Night of the Dead",
+    "features": null,
+    "images": [
+        "quay.io\/parkervcp\/pterodactyl-images:debian_wine-5"
+    ],
+    "file_denylist": [],
+    "startup": "cp \/home\/container\/notd\/ServerSettings.ini \/home\/container\/notd\/LF\/Saved\/Config\/ServerSettings.ini;\r\ncp \/home\/container\/notd\/steamclient64.dll \/home\/container\/notd\/LF\/Binaries\/Win64\/steamclient64.dll;\r\ncp \/home\/container\/notd\/tier0_s64.dll \/home\/container\/notd\/LF\/Binaries\/Win64\/tier0_s64.dll;\r\ncp \/home\/container\/notd\/vstdlib_s64.dll \/home\/container\/notd\/LF\/Binaries\/Win64\/tier0_s64.dll;\r\ncd \/home\/container\/notd\/; \r\nwine \/home\/container\/notd\/LF\/Binaries\/Win64\/LFServer.exe ?listen -log -Port=7777 -QueryPort=27015 -CRASHREPORTS",
+    "config": {
```

Comment:
I know, that the StartServer.bat also copies this 4 files at every start. But I think it would be sufficient to put this cp commands in your install script. Also don't forget to use variables here.
And don't forget to correct it to
```
cp /home/container/notd/tier0_s64.dll /home/container/notd/LF/Binaries/Win64/tier0_s64.dll;
cp /home/container/notd/vstdlib_s64.dll /home/container/notd/LF/Binaries/Win64/vstdlib_s64.dll;
```

Reply:
Okey altho i dont see a difference between:
cp /home/container/notd/tier0_s64.dll /home/container/notd/LF/Binaries/Win64/tier0_s64.dll;
and
cp /home/container/notd/tier0_s64.dll /home/container/notd/LF/Binaries/Win64/tier0_s64.dll;

I did however change:
cp /home/container/notd/vstdlib_s64.dll /home/container/notd/LF/Binaries/Win64/tier0_s64.dll;
since its incorrect.